portfolio/src/main/webapp/index.html

```diff
@@ -42,7 +42,8 @@ <h1>Tal's Portfolio</h1>
         <p>Post a comment here as:</p>
         
         <!--stores email from user server in this form to send to data server-->
-        <p name = "email" id ="poster"></p>
+        <input name = "email" id ="poster" value="" readonly />
```

Comment:
Is there a way to make this have its own line/not have the input box around it?

Reply:
Yes, css should allow you to remove the border + change the background color